### PR TITLE
Fix live bugs reported in FB

### PR DESCRIPTION
**Pull request by ErkoRisthein**

### CODE REVIEW

File: src/account/AccountPage.js [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
     <div className="row mt-5">
       <div className="col">
-        <Message params={{ memberNumber }}>account.member.statement</Message>
+        { memberNumber ?
+          <Message params={{ memberNumber }}>account.member.statement</Message> :
+          (
+            <span>
+              <Message>account.non.member.statement</Message>
+              { ' ' }
+              <Link className="btn btn-link p-0 border-0" to="/steps/new-user">
```
Comment (JordanValdma) [inline src/account/AccountPage.js]: test?

File: src/account/AccountPage.js [hunk 2]
```diff
@@ -34,7 +48,11 @@ export const AccountPage = ({
     }
     <div className="row mt-5">
       <div className="col">
-        <PensionFundTable funds={currentBalanceFunds} />
+        {
```
Comment (JordanValdma) [inline src/account/AccountPage.js]: test?

File: src/account/AccountPage.js (hunk 1 above)
Comment (ErkoRisthein) [inline src/account/AccountPage.js]: 👮

File: src/account/AccountPage.js (hunk 2 above)
Comment (ErkoRisthein) [inline src/account/AccountPage.js]: 👮

**Pull request merged**